<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Items</class>
 <widget class="QMainWindow" name="Items">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Chickenlabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Chicken Options</string>
    </property>
   </widget>
   <widget class="QLabel" name="Meatlabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Meat Options</string>
    </property>
   </widget>
   <widget class="QLabel" name="ItemPriceLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Item Price</string>
    </property>
   </widget>
   <widget class="QLabel" name="NumberofItemsLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of added items</string>
    </property>
   </widget>
   <widget class="QLabel" name="BalanceLabel2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Account Balance: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddtoCartButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>170</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Add to Cart</string>
    </property>
   </widget>
   <widget class="QLabel" name="PriceLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="numberitemsLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Balanceval">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="CHICKENcomboBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="MEATcomboBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="viewCartButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>ViewCart</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
